braille.txt

```diff
@@ -0,0 +1 @@
+hello world
```

Comment:
test code for iteration 1 test to pass, what is being read and then translated (just English to English for now)